pitop/camera/camera.py

```diff
@@ -32,10 +32,12 @@ class Camera(Stateful, Recreatable):
 
     def __init__(self,
                  index=None,
-                 resolution=None,
+                 resolution=(640, 480),
                  camera_type=CameraTypes.USB_CAMERA,
                  path_to_images="",
                  format='PIL',
+                 flip_top_bottom: bool = False,
+                 flip_left_right: bool = False,
```

Comment:
No.

Reply:
https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.transpose - it's needed to flip a camera frame from top to bottom or left to right. Prax's camera is mounted in a flipped orientation and because it's facing the user it needs to be flipped (mirrored) and not just rotated

Reply:
added to description